Packs/IllusiveNetworks/Integrations/IllusiveNetworks/IllusiveNetworks.yml

```diff
@@ -0,0 +1,521 @@
+  name: proxy
+  required: false
+  type: 8
+- defaultvalue: 3 days
+  display: The initial time to fetch from
+  hidden: false
+  name: fetch_time
+  required: false
+  type: 0
+description: The Illusive Attack Management API allows customers to retrieve detected
+  incidents with a forensics timeline, attack surface insights, collect forensics
+  on-demand, and manage a variety of operations with regard to deceptive entities,
+  deception policies, and more.
+display: IllusiveNetworks
+name: IllusiveNetworks
+script:
+  commands:
+  - arguments:
+    - default: false
+      description: The desired incident ID
+      isArray: false
+      name: incident_id
+      required: true
+      secret: false
+    - default: false
+      description: |-
+        The starting date of the forensics timeline.
+        start_date is in Zulu time format, for example: 1993-09-24T17:30:00.000Z.
+      isArray: false
+      name: start_date
+      required: false
+      secret: false
+    - default: false
+      description: |-
+        The last date of the forensics timeline.
+        end_date is in Zulu time format, for example: 1993-09-24T17:30:00.000Z.
+      isArray: false
+      name: end_date
+      required: false
+      secret: false
+    deprecated: false
+    description: Retrieve forensics timeline for a specific incident
+    execution: false
+    name: illusive-get-forensics-timeline
+    outputs:
+    - contextPath: Illusive.Forensics.Evidence.details
+      description: The forensics evidence details
+      type: String
+    - contextPath: Illusive.Forensics.Evidence.eventId
+      description: The event ID
+      type: String
+    - contextPath: Illusive.Forensics.Evidence.id
+      description: The forensics evidence ID
+      type: String
+    - contextPath: Illusive.Forensics.Evidence.source
+      description: The Evidence source
+      type: String
+    - contextPath: Illusive.Forensics.Evidence.starred
+      description: Whether the forensics evidence has been starred
+      type: Boolean
+    - contextPath: Illusive.Forensics.Evidence.time
+      description: 'Date and time of the forensics evidence '
+      type: Date
+    - contextPath: Illusive.Forensics.Evidence.title
+      description: The forensics evidence description
+      type: String
+    - contextPath: Illusive.Forensics.IncidentId
+      description: The Incdient Id
+      type: String
+    - contextPath: Illusive.Forensics.Status
+      description: The process progress ( Done, InProgress)
+      type: String
+  - arguments:
+    - default: false
+      description: The hostname or IP address of the desired host
+      isArray: false
+      name: hostnameOrIp
+      required: true
+      secret: false
+    deprecated: false
+    description: Retrieve the specified host insights from Attack Surface Manager
+    execution: false
+    name: illusive-get-asm-host-insight
+    outputs:
+    - contextPath: Illusive.AttackSurfaceInsights.Host.DomainName
```

Comment:
I would suggest to change the context output path to be like `Illusive.AttackSurfaceInsightsHost.DomainName`
The reason for that is currently in Demisto filtering outputs more that 3 level nested is a bit more hard.